Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/csiaddons/k8s-controller:latest
```

Comment:
the image name in deployment artifacts need to be updated with new name

Reply:
where? `make deploy` seems to work with this change, but not with the original name

Reply:
okay `make deploy` edit the image name before applying